README.rst

```diff
@@ -217,71 +217,48 @@ You can change this by using `use_bin_type=True` option in Packer and `encoding=
     ['spam', u'egg']
 
 It's worth noting that ``encoding`` (for ``packb()`` and ``unpackb()`` alike), ``use_bin_type``, and other
-parameters have nothing to do with encoding of the string representation of the packed string itself,
-but rather with the encoding of the final unpacked item.
-
-For example:
+parameters have nothing to do with encoding of the packed representation of the string itself,
+but rather with the encoding of the final unpacked item. In other words, the packed representation
+of the item is always going to be a string, seemingly encoded using `latin1` encoding:
```

Comment:
msgpack is "binary" format.  It's not "latin1 encoded string".  Just a binary.  Not a string.